src/lib/coreboot_table.c

```diff
@@ -408,6 +410,45 @@ void __weak lb_board(struct lb_header *header) { /* NOOP */ }
  */
 void __weak lb_spi_flash(struct lb_header *header) { /* NOOP */ }
 
+/*
```

Comment:
Have you seen clevo/release implementation for the same thing? It was updated for the newer CBFS API and avoids manual buffer allocation.

https://github.com/Dasharo/coreboot/blob/ae10b20f5c6abc9c23f709b65c46be6525da8c13/src/lib/coreboot_table.c#L408-L434

Reply:
I can rewrite it to such state.

Reply:
@mkopec done

Reply:
At this point it made more sense to simply cherry-pick your commit.

Reply:
Thanks